pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.confluent</groupId>
+        <artifactId>common</artifactId>
+        <version>5.1.0-SNAPSHOT</version>
+    </parent>
+
+    <groupId>io.confluent</groupId>
+    <artifactId>cp-docker-images-parent</artifactId>
+    <packaging>pom</packaging>
+    <name>cp-docker-images-parent</name>
+    <version>5.1.0-SNAPSHOT</version>
+
+    <licenses>
+        <license>
+            <name>Apache License 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.html</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <modules>
+        <module>docker-utils</module>
```

Comment:
We will need to version bump if we want to integrate into the rest of the pipeline.
Currently, the pom is within a folder docker-utils (previously: java) and going through the code it appeared that it wouldn't version bump. Hence I added a parent pom so that the version bump can follow the pom (looking at how ksql and others similar repos are setup).

Not sure if there is a better way to do it.

Reply:
For single module, the pom should be at the top level, though I thought our version bump code just looks thru all pom.xml recursively so it shouldn't matter where it is. See license-file-generator

If our version bump code isn't working for a project without parent/mult-module setup, that feels like a bug that needs to be fixed as it should work for single module poms.

Reply:
I think I know why it didn't work. It's because _get_version_mvn is failing as there wasn't a top level pom (root level) as that method only looks at the root project dir -- which is where it should be. https://github.com/confluentinc/release-tools/blob/master/confluent/release/versions.py#L70

Reply:
This is what I was trying to do:
https://jenkins.confluent.io/job/confluentinc/job/tools-autotasks/job/apply-nightly-tags/22/console

tbr --debug apply-release-tag --release-quality beta --release-number 180701205239 --confluent-patch 1 --build release --env-name prod cp-docker-images --dry-run

Reply:
Yup .

Reply:
@maxzheng should i move the pom and the src structure to the root folder? I think read me is irrelevant at this point since the code related to it no longer exists.